src/Tribe/Main.php

```diff
@@ -3743,8 +3743,10 @@ public function get_event_link( $post, $mode = 'next', $anchor = false ) {
 					$anchor = preg_replace( '|%title%|', $title, $anchor );
 				}
 
-				return apply_filters( 'tribe_events_get_event_link', '<a href="' . esc_url( tribe_get_event_link( $results ) ) . '">' . $anchor . '</a>' );
+				$link = '<a href="' . esc_url( tribe_get_event_link( $results ) ) . '">' . $anchor . '</a>';
 			}
+
+			return apply_filters( 'tribe_events_get_event_link', $link );
```

Comment:
Yes, you're right - I was actually toying with providing the function arguments - `$post`, `$mode` and `$anchor` - not sure `$args` is needed or the right fit for this hook (since it can be modified in an earlier filter) - what do you think?